From 56524bfbc676de2e4f92f649271c6a776dde7ad8 Mon Sep 17 00:00:00 2001
From: Chris Castillo <chris.castillo@webpros.com>
Date: Tue, 14 Oct 2025 15:20:22 +0000
Subject: [PATCH] Fix libxml2 v2.15.0 compatibility

---
 ext/libxml/libxml.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/libxml/libxml.c b/ext/libxml/libxml.c
index 85f239d5c72..ec08ad9867c 100644
--- a/ext/libxml/libxml.c
+++ b/ext/libxml/libxml.c
@@ -428,7 +428,7 @@ php_libxml_input_buffer_create_filename(const char *URI, xmlCharEncoding enc)
 static xmlOutputBufferPtr
 php_libxml_output_buffer_create_filename(const char *URI,
                               xmlCharEncodingHandlerPtr encoder,
-                              int compression ATTRIBUTE_UNUSED)
+                              int compression)
 {
 	xmlOutputBufferPtr ret;
 	xmlURIPtr puri;
-- 
2.48.2

